### PR TITLE
[hi] Update Hindi

### PR DESCRIPTION
**Pull request by nishadil**
Following key for "hi" lang in php.json updated 

accepted_if , 
declined,
declined_if,
enum,
mac_address,
password.letters

more will be done soon...

### CODE REVIEW

File: locales/hi/php.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "lte.file": "The :attribute must be less than or equal :value kilobytes.",
     "lte.numeric": "The :attribute must be less than or equal :value.",
     "lte.string": "The :attribute must be less than or equal :value characters.",
-    "mac_address": "The :attribute must be a valid MAC address.",
+    "mac_address": ":Attribute एक मान्य MAC address होना चाहिए.",
```
Comment (andrey-helldar) [inline locales/hi/php.json:64]: Here, the application should definitely end with a full stop, and not the `।` character?

Comment (andrey-helldar) [inline locales/hi/php.json:64]: Is it correct that the sentence ends with a full stop and not the `|` symbol?

Comment (nishadil) [reply locales/hi/php.json:64]: You are right. In Hindi language, sentence should be ends with `।` symbol. sorry for this silly typo-mistake . Lemme fix it.

Comment (andrey-helldar) [reply locales/hi/php.json:64]: Make changes, commit and push. So these changes will fall into this PR.

**Pull request closed without merging**